src/Directory.Build.props

```diff
@@ -21,8 +21,12 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
   </ItemGroup>
 
+    <ItemGroup>

```

Comment:
nit: indentation.
Otherwise, I'm thrilled. Don't send a fixup. Just squash locally or amend or whatever.

Reply:
Sorry! I made more work for you!

Reply:
Oops. Will fix.

Reply:
Done